Account/FirefoxAccount.swift

```diff
@@ -139,7 +139,10 @@ open class FirefoxAccount {
 
     open class func fromDictionary(_ dictionary: [String: Any]) -> FirefoxAccount? {
         if let version = dictionary["version"] as? Int {
-            if version == AccountSchemaVersion {
+            // As of this writing, the current version, v2, is backward compatible with v1. The only
+            // field added is pushRegistration, which is ok to be nil. If it is nil, then the app
+            // will attempt registration when it starts up.
+            if version <= AccountSchemaVersion {
                 return FirefoxAccount.fromDictionaryV1(dictionary)
             }
```

Comment:
Happy with this code for this release, but `fromDictionaryV1` should be tied directly to `version` being `1` or `2`, rather than a monotonically increasing `AccountSchemaVersion`.